pkgs/development/tools/misc/nix-build-uncached/default.nix

```
@@ -0,0 +1,30 @@
+{ lib, buildGoPackage, fetchFromGitHub, nix, makeWrapper }:
+
+buildGoPackage rec {
```

```suggestion
buildGoModule rec {
```

and then set `vendorSha256 = null;`